platform_plugin_forum_email_notifier/utils.py

```diff
@@ -10,6 +12,23 @@
 from platform_plugin_forum_email_notifier.models import ForumNotificationPreference, PreferenceOptions
 
 
+def get_simplified_text(text: str) -> str:
+    """
+    Return the simplified text of a html string.
+
+    If the text is longer than 160 characters, it will be truncated.
+
+    Args:
+        text (str): The html string.
+
+    Returns:
+        str: The simplified text.
+    """
+    soup = BeautifulSoup(text, "html.parser")
+    text = soup.get_text()
+    return text if len(text) <= 160 else f"{text[:160]}..."
```

Comment:
I would create a function here with the max size allowed and suffix 
```
def truncate_text(text, max_length=160, suffix="..."):
    if len(text) <= max_length:
        return text
    return f"{text[:max_length]}{suffix}"
```
then
```
soup = BeautifulSoup(text, "html.parser")
text = soup.get_text()
return self.truncate_text(text, 160)
```

Reply:
Done